README.md

```diff
@@ -49,7 +49,9 @@ The following files need to be submitted:
 * ``main.py``
 * ``test_main.py``
 * ``menu.py``
+* ``test_menu.py``
```

Comment:
Have you had your students add testing for _menu.py_? One of the concerns about asking for this was to create too much overhead for a single assignment, with students having to implement a SQL database and test it.

Reply:
Oh, actually, I am not. Let me remove this line